packages/app-project/src/screens/ProjectHomePage/components/Hero/helpers/fetchWorkflowsHelper/fetchWorkflowsHelper.js

```diff
@@ -9,7 +9,7 @@ async function fetchWorkflowData (activeWorkflows) {
       include: 'subject_sets'
     })
   const { workflows, linked } = response.body
-  const subjectSets = linked.subject_sets
+  const subjectSets = linked ? linked.subject_sets : []
```

Comment:
Non-blocking, but this can also be written now as `const subjectSets = linked?.subject_sets ?? []` since the optional chaining operator and the nullish coalescing operator are stage 4 (last section in the MDN article has an example of them used together: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_Coalescing_Operator).